src/docs/asciidoc/distributed_sql.adoc

```diff
@@ -0,0 +1,211 @@
+
+
+== Distributed SQL
+
+Distributed SQL allows you to access data stored in the Hazelcast cluster
+declaratively using SQL.
+
+=== Beta Status
+
+The distributed SQL is currently in beta. The compatibility between versions
+is not guaranteed. API might change between versions without notice.
+
+=== How Distributed SQL works
```

Comment:
In my opinion, this section is very technical and low level and is not actually useful for the end user. I know that this is not the final documentantation, but let's write even the small piece from the user perspective on the first place.

Therefore, I don't have anything against describing how it works, but this section should be at the end of the chapter. Kind of like interesting details.

Reply:
Please move the example of how to execute SQL query right here to the very beginning. Possibly with a caption "How to execute SQL query".

Reply:
Moved example to the top, "How Distributed SQL Works" to the bottom